ruff_cli/src/printer.rs

```diff
@@ -312,6 +312,20 @@ impl<'a> Printer<'a> {
                     )?
                 )?;
             }
+            SerializationFormat::Pylint => {
+                // Generate violations in Pylint format.
+                // See: https://flake8.pycqa.org/en/latest/internal/formatters.html#pylint-formatter
+                for message in &diagnostics.messages {
+                    let label = format!(
+                        "{}:{}: [{}] {}",
```

Comment:
Tweaked this to put some of the plain-text characters in the format string directly.

Reply:
I was unsure about this but this is more what I would expect.